docker-compose.yml

```diff
@@ -25,80 +25,116 @@ services:
       - "./docker/configs/elasticsearch:/usr/share/elasticsearch/config"
     restart: unless-stopped
 
-  # Base container for sbt-based services.
```

Comment:
I was being stupid in https://github.com/guardian/grid/pull/1730 - the `sbt` container shouldn't be in the main compose file!